src/components/SecondaryPanes/Frames/tests/FrameMenu.js

```diff
@@ -0,0 +1,73 @@
+const originalL10N = L10N;
+import FrameMenu from "../FrameMenu";
+import { showMenu } from "devtools-launchpad";
+import { copyToTheClipboard } from "../../../../utils/clipboard";
+jest.mock("devtools-launchpad", () => ({ showMenu: jest.fn() }));
+jest.mock("../../../../utils/clipboard", () => ({
+  copyToTheClipboard: jest.fn()
+}));
+
+function generateMockCopySource(label, accesskey) {
```

Comment:
Hmm... if the formatting was taken care of elsewhere, I would agree that relying on the IDs is a much better pattern, but since `FrameMenu` is the responsible function, I think we should test that we get correctly formatted output. Testing the id's is insufficient, because if something is changed in FrameMenu, ie, someone mixes up the label and the key, we will get the wrong output -- and this would be hard to catch elsewhere.

However as you mentioned, this could be moved out into a utility -- if that happens, this level of testing should move with it!